### DISCUSSION TITLE
How to configure host system to use dms for delivery?

### DISCUSSION THREAD

**Discussion by jakkarth** [docker-mailserver/docker-mailserver#4284] [Q&A] [answered]
I've got docker mailserver running in docker on my host. All of that's working for mail going to and from, and it's great. With one exception: mail generated on the host itself doesn't get delivered anywhere.

For example, running

```
echo 'this is my message' | mail myuser@mydomain.com -s 'test message'
```

never arrives. Obviously I can't install postfix on the host to handle delivery because all those ports are open by DMS. So, how do I configure the host to be able to use DMS running in docker *on* the host to get delivery working?

**Comment by casperklein** [comment, marked as answer]
You can use [nullmailer](https://wiki.debian.org/nullmailer) for that purpose. It can forward all local mails to docker-mailserver and doesn't require to bind port 25.
